docs/settings/security-settings.asciidoc

```diff
@@ -26,6 +26,28 @@ screens, and authorization using <<kibana-privileges>>. To disable
 Set to `true` to enable audit logging for security events. By default, it is set
 to `false`. For more details see <<xpack-security-audit-logging>>.
 
+[float]
+[[authentication-security-settings]]
+==== Authentication security settings
+
+You configure the authentication settings in the `xpack.security.authc.providers` namespace. For more information, refer to <<kibana-authentication>>.
+
+`xpack.security.authc.providers`::
+Identifies the authentication method.
```

Comment:
__question:__ can we maybe write something similar to [what we have currently](https://www.elastic.co/guide/en/kibana/current/security-settings-kb.html#CO6-1)? Something like:
```ascii
Specifies the types of authentication providers.
```

-------------

`The SAML realm in {es} that the provider should use.` - this one is allowed and makes sense only if we have `saml` string in `xpack.security.authc.providers` array. Can we say ``The SAML realm in {es} that the `saml` provider should use, if specified in `xpack.security.authc.providers`.`` or something similar that's more grammatically correct :slightly_smiling_face: 

-------------

`The OpenID Connect realm in {es} that the provider should use.` - this one is allowed and makes sense only if we have `oidc` string in `xpack.security.authc.providers` array. Can we say ``The OpenID Connect realm in {es} that the `oidc` provider should use, if specified in `xpack.security.authc.providers`.``